scss/components/_off-canvas.scss

```diff
@@ -417,7 +417,7 @@ $breakpoint: small
   height: auto;
   position: static;
   background: inherit;
-  width: inherit;
+  width: auto;
```

Comment:
This makes sense for `width: auto`, but @SassNinja was there any reasons for going for `inherit` in the first place for properties that does not inherit their value by default ?

Reply:
poke @SassNinja

Reply:
Done some research. The incanvas feature always used `inherit`, without explicit reasons given. Most font properties should inherit but not `background`, `width`, `overflow`, `transition` as this is not their default value.

Reply:
@ncoden the in-canvas is supposed to just revert the off-canvas styles. Seems I was overusing the `inherit` a bit ;)
You're replacements of the properties in talk look good to me 👍